Makefile

```diff
@@ -63,9 +61,7 @@ test-coverage-html:
 	go tool cover -html .cover/cover.out
 
 dist: ## Package up everything in static/ using go-bindata-assetfs so it can be served by a single binary
-	rm -f server/static/bindata_assetfs.go && \
-	go-bindata-assetfs -o server/static/bindata_assetfs.go -pkg static -prefix server server/static/... && \
-	go fmt server/static/bindata_assetfs.go
+	rm -f server/static/bindata_assetfs.go && go-bindata-assetfs -pkg static -prefix server server/static/... && mv bindata_assetfs.go server/static
```

Comment:
I think this is wrong. On my computer, the output of `go-bindata-assetfs` was not `bindata_assetfs.go`. I think it was `bindata.go`. The version you are replacing looks better to me, explicitly stating the output file name.

Reply:
this is 
```
rm -f server/static/bindata_assetfs.go && go-bindata-assetfs -pkg static -prefix server server/static/... && mv bindata_assetfs.go server/static
```

is from the original repo

and it does not specify any outputs (-o param).
if it does nor specify any output, then `go-bindata` generates `bindata.go`.
But the make target uses `mv bindata_assetfs.go server/static` which does not work
So I think the original target (which I copied back to reduce our changes) is wrong (or they use something else to do it)

Reply:
just reversed it back to reduce our changes